.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "consistent-return": "off",
+    "no-useless-escape": "off"
```

Comment:
I've had false positives with this rule in the past... I won't modify the RegExp on the email function.